metadata.json

```diff
@@ -78,7 +78,7 @@
       "version_requirement": ">= 5.5.10 < 7.0.0"
     }
   ],
-  "template-url": "https://github.com/puppetlabs/pdk-templates#main",
-  "template-ref": "heads/main-0-g874030e",
+  "template-url": "https://github.com/puppetlabs/pdk-templates#adrianiurca-allow-use-dist-locally",
+  "template-ref": "remotes/origin/adrianiurca-allow-use-dist-locally-0-g55e608a",
```

Comment:
This shouldn't be the reference we're using - this was just for testing. Can you create another PR to resolve this by just running `pdk update` without the `--template-ref` option after https://github.com/puppetlabs/pdk-templates/pull/366 has been merged